docs/source/examples.rst

```diff
@@ -7,18 +7,19 @@ Usage Examples
    :maxdepth: 1
 
 .. _example_dependencies:
+
 Example Dependencies
-------------------------
+--------------------
 
-Some examples may use additional dependecies. These dependencies are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
-Pip should be used to install these extra requirements except astra which should be installed via conda:
+Some examples use additional dependencies which are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
+The additional requirements should be installed via pip, with the exception of astra-toolbox, which should be installed via conda:
```

Comment:
Nitpick: `astra-toolbox` (or is it ``astra-toolbox`` for consistency with usage elsewhere in the docs?)